py/torch_tensorrt/dynamo/conversion/converter_utils.py

```diff
@@ -188,3 +188,17 @@ def extend_attr_to_tuple(
     if isinstance(val, list):
         val = tuple(val)
     return val
+
+
+def trt_cast_int_to_float(network: TRTNetwork, name: str, tensor: TRTTensor):
+    if tensor.dtype == trt.int8 or tensor.dtype == trt.int32:
```

Comment:
I think this should only apply to `trt.int32`, since `trt.int8` is a special case (quantization) which seems to be supported by all elementwise ops except the boolean ones.